schedule/schedule-appengine/src/main/client/jasify/booking-via-jasify/activity-packages.selection.html

```diff
@@ -30,22 +30,22 @@ <h2 class="lvh-label">{{activityPackage.name}}</h2>
 
                 <h2 class="lvh-label">
                     <small>
-                        Select <strong>{{activityPackage.itemCount}} of the
-                        {{vm.activityPackageActivities[activityPackage.id].length}}</strong> activities
+                        Select <strong>{{activityPackage.itemCount}}</strong> of the
+                        <strong>{{vm.activityPackageActivities[activityPackage.id].length}}</strong> activities
                         below.
                     </small>
                 </h2>
                 <h2 class="lvh-label">
-                    <small>Price: <strong>{{activityPackage.price}}
+                    <small>{{ 'Price' | translate }}: <strong>{{activityPackage.price}}
                         {{activityPackage.currency}}</strong></small>
                 </h2>
                 <h2 class="lvh-label m-r-10">
                                 <span ng-show="vm.packageSelectionIncomplete(activityPackage)"
                                       class="label label-warning">
-                                    {{vm.activityPackageSelection[activityPackage.id].length || 0}}/{{activityPackage.itemCount}} Activities selected.</span>
+                                    {{vm.activityPackageSelection[activityPackage.id].length || 0}}/{{activityPackage.itemCount}} {{ 'ACTIVITEIS_SELECTED' | translate }}</span>
                                 <span ng-show="vm.packageSelectionComplete(activityPackage)"
                                       class="label label-info">
-                                    Package selection completed.</span>
+                                    {{ 'SELECTION_COMPLETED' | translate }}</span>
                                 <span ng-show="vm.packageSelectionTooBig(activityPackage)"
                                       class="label label-info">
                                     You selected to many activities. Exactly {{activityPackage.itemCount}} activities must be selected.</span>
```

Comment:
This one is pending. I had a problem with dynamic values and formatting and have aske Pawel for help. Have raised issue for it so I dont forget, #608